README.md

```diff
@@ -37,3 +18,18 @@ mvn -s /path/to/maven-settings.xml exec:exec
 
 Note that the `maven-settings.xml` file is available in the root directory of
 this repository.
+
+## Remote cache tutorials
+
+Tutorials that use remote caches require at least one Data Grid Server running locally.
+
+Download the server distribution from the [Red Hat customer portal](https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=data.grid&downloadType=distributions) and run:
+
+```bash
+./bin/cli.sh user create admin -p "changeme"
```

Comment:
the password is hardcoded. you have to use `password` :)

Reply:
bah! I forgot.